### PR TITLE
Support binding types that implement text encoding

### PR DESCRIPTION
**Pull request by devnev**
I ran across this interface while trying to create a flag from a logrus.Level values, and thought it might make sense to support it directly in flagbind.

### CODE REVIEW

**Comment by devnev** [comment]
ping @AdamSLevy

**Comment by devnev** [comment]
ping @AdamSLevy, any concerns about merging this?

File: bind.go [hunk 1]
```diff
@@ -561,6 +562,11 @@ func bindSTDFlag(fs STDFlagSet, tag flagTag, p interface{}) bool {
 	case *string:
 		val := *p
 		fs.StringVar(p, tag.Name, val, tag.Usage)
+	case textBidiMarshaler:
+		// Match the interface after concrete types so that any concrete types that
+		// also implement the interface use the more specific implementation for
+		// their concrete types.
```
Comment (AdamSLevy) [inline bind.go:568]: Great consideration.

**Review by AdamSLevy — APPROVED** [review]
LGTM

Thank you for the contribution! Please feel free to open issues if you have ideas for larger features or improvements. 

Also feel free to email me if you want to connect. adam at aslevy dot com. I'm very curious to hear about how you're using this package and how you came across it.

File: bind.go (hunk 1 above)
Comment (devnev) [reply bind.go:568]: Found based on existing tests, but making the rationale clear in a comment seemed like it'd be helpful

**Pull request merged**